### ISSUE TITLE
No feedback if you execute a mutation for a cyclepoint/task that is gone from the UI

### ISSUE THREAD

**Issue by kinow** [cylc/cylc-ui#639]
**Describe the bug**

At the moment you can click on items in the UI tree to display a mutations form. The form contains entries that you can then click to trigger mutations in the backend.

If a task or cyclepoint is short-lived, or if it is about to disappear, the mutations form is not removed.

And if you click on Hold for example, there is no feedback in the UI, nor in the browser console. In the Cylc UIS logs I can see a warning saying that the task does not exist. But it would be nice to have some visual feedback.

**Release version(s) and/or repository branch(es) affected?**

`master`

**Steps to reproduce the bug**

See description.

**Expected behavior**

A visual feedback.

**Screenshots**

**Additional context**

I think we have a related issue here or in Cylc UIS, but I couldn't find it, so logging this new one so I don't forget it :+1: 

**Pull requests welcome!**
This is an Open Source project - please consider contributing a bug fix
yourself (please read `CONTRIBUTING.md` before starting any work though).

**Comment by hjoliver** [comment]
This is a tricky one.

Part of the solution is getting feedback from the scheduler about command completion rather than just the "command queued" message  (I think we have an issue for that somewhere), because for some commands it doesn't actually matter if the task is gone from the UI.  For commands where that does matter, we could presumably have a UI-driven solution.

**Comment by oliver-sanders** [comment]
I think:

* If the mutation menu (the thing that lists the mutations) is open when the thing that opened it (e.g. the cycle point) moves out of the window then it makes sense to detect this and close the mutation menu. This menu should be short lived as we would expect users to select a mutation or to close it.
* If the mutation editor is open we should leave it as the workflows/cycles/families/tasks/jobs that the mutation is called against may be edited from that form.
